z2/resources/chain-specs/zq2-devnet.toml

```diff
@@ -5,7 +5,7 @@ bootstrap_address = [ "12D3KooWJNUNgs1UM9mw1Pfskxk4o3VWz37cpneSHZsUecGGZgkb", "/
 eth_chain_id = 33469
 allowed_timestamp_skew = { secs = 60, nanos = 0 }
 data_dir = "/data"
-consensus.genesis_accounts = [ ["0x977a797a0546cdb1082c2368921af455dfdc7a5a", "21_000_000_000_000_000_000_000_000_000"] ]
+consensus.genesis_accounts = [ ["0x977a797a0546cdb1082c2368921af455dfdc7a5a", "210_000_000_000_000_000_000_000_000"] ]
```

Comment:
changing the value in the config.tera.toml may be worth changing the value also in all the chain specs except for prototestnet. The reason is because the devnets are deployed quite often. Does this make sense?

Reply:
OK - again, separate PR else will never merge.

Reply:
Fixed in https://github.com/Zilliqa/zq2/pull/1683 (I hope!)